google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/CloudSdkAppEngineHelper.java

```diff
@@ -137,7 +137,7 @@ public AppEngineDeployAction createAutoDeploymentAction(
         artifactToDeploy,
         defaultAppYaml(),
         defaultDockerfile(artifactType),
-        AppEngineUtil.generateVersion(),
+        version,
```

Comment:
The duplication between `createCustomDeploymentAction` and `createAutoDeploymentAction`?